<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WriteOnChain</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Write your message</string>
    </attribute>
    <widget class="QLineEdit" name="txt_message">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>611</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_chain">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Blockchain</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_chain">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>221</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>BTC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>tBTC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LTC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>tLTC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LIQUID</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lbl_result">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>Results</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_write">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>411</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Write</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txt_result">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>611</width>
       <height>121</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.SF NS Text';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Configuration</string>
    </attribute>
    <widget class="QLineEdit" name="txt_BTC_host">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_BTC_port">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>8332</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_chain_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>39</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>BTC</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_BTC_username">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_BTC_password">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_BTC_wallet">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>wallet.dat</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_chain_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Host</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_chain_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_chain_5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_chain_6">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_chain_7">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Wallet</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_tBTC_host">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_LTC_host">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_tLTC_host">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_LIQUID_host">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_chain_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>tBTC</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_chain_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>LTC</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_chain_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>tLTC</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_chain_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Liquid</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_tBTC_port">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>18332</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_LTC_port">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>9332</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_tLTC_port">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>19332</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_LIQUID_port">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>7041</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_tBTC_username">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_LTC_username">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_tLTC_username">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_LIQUID_username">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_tBTC_password">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>70</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_LTC_password">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>100</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_tLTC_password">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>130</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_LIQUID_password">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>160</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_tBTC_wallet">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>70</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>wallet.dat</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_LTC_wallet">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>100</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>wallet.dat</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_tLTC_wallet">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>130</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>wallet.dat</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_LIQUID_wallet">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>160</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>wallet.dat</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_config_load">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>241</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_config_save">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>190</y>
       <width>241</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>save</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_BTC_passphrase">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_chain_12">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Passphrase</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_tBTC_passphrase">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>70</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_LTC_passphrase">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>100</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_tLTC_passphrase">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>130</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_LIQUID_passphrase">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>160</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
